package.json

```diff
@@ -30,5 +30,8 @@
   },
   "jest": {
     "testEnvironment": "jsdom"
+  },
+  "engines": {
+    "node": "<15.4"
```

Comment:
Since Node v14 is in maintenance mode and v15 is no longer supported, I'd rather we set this to `14.x`

```suggestion
    "node": "14.x"
```

Reply:
<img width="434" alt="image" src="https://user-images.githubusercontent.com/229881/213000022-d9a752a2-ea6f-449a-afad-cad610e548a6.png">


Probably `<14.17.0` would be even more accurate